core/templates/dev/head/pages/exploration_editor/editor_tab/StateHints.js

```diff
@@ -51,6 +75,40 @@ oppia.controller('StateHints', [
       return hintAsPlainText;
     };
 
+    $scope.getQuittersPercent = function(currentHintIndex) {
+      var currentHintStats = null;
+      var nextHintStats = null;
+      if (currentHintIndex < stateHintsService.savedMemento.length - 1) {
+        currentHintStats = $scope.hintViewsStats.hints[currentHintIndex];
+        if ($scope.hintViewsStats.hints[currentHintIndex + 1]) {
```

Comment:
It's unclear to me whether this is an === 0 check, or a length check. Might be better to be explicit, e.g. using .length for the latter.